src/hooks/http.ts

```diff
@@ -238,13 +306,22 @@ function handleResponse(
   startTime: number,
   timestamp: string | undefined,
   response: http.ServerResponse<http.IncomingMessage>,
+  chunks: Chunk[],
 ): void {
   if (fixupEvent(request, requestEvent)) recording.fixup(requestEvent);
+
+  let body =
+    chunks.length > 0 && typeof chunks[0] === "string"
+      ? chunks.join("")
+      : Buffer.concat(chunks as Buffer[]).toString("utf8");
+  body = body.substring(0, config.responseBodyMaxLength);
+
   recording.httpResponse(
     requestEvent.id,
     getTime() - startTime,
     response.statusCode,
     normalizeHeaders(response.getHeaders()),
+    body ? { class: "[ResponseBody]", value: body } : undefined,
```

Comment:
If the _Content-Type_ header indicates JSON, we should attempt to parse it and just hand it to `parameter` if successful. This will allow generation of OpenAPI data based on this.